pkg/webui/console/components/gateway-api-keys-modal/index.js

```diff
@@ -0,0 +1,89 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import PortalledModal from '@ttn-lw/components/modal/portalled'
+import Button from '@ttn-lw/components/button'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import style from './gateway-api-key-modal.styl'
+
+const m = defineMessages({
+  modalTitle: 'Download gateway API keys',
+  buttonMessage: 'I have downloaded the keys',
+  description:
+    'Note: After proceeding to the gateway, the API keys will not be accessible for download anymore. Make sure to download and store them.',
```

Comment:
```suggestion
    'Note: After closing this window, these API keys will not be accessible for download anymore. Please make sure to download and store them now.',
```

Sorry but have to reword this a bit more.